src/analysis/convex/strict_convex_space.lean

```diff
@@ -0,0 +1,127 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Yury Kudryashov
+-/
+import analysis.convex.strict
+import analysis.convex.topology
+import analysis.normed_space.ordered
+import analysis.normed_space.pointwise
+
+/-!
+# Strictly convex spaces
+
```

Comment:
Maybe add a main defintions and main results sections? I guess `dist_add_dist_eq_iff` is the important property